tests/framework/installer/ceph_installer.go

```diff
@@ -366,7 +366,7 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(helmInstalled bool, systemNa
 		checkError(h.T(), err, fmt.Sprintf("cannot remove cluster %s", namespace))
 
 		crdCheckerFunc := func() error {
-			_, err := h.k8shelper.RookClientset.RookV1alpha1().Clusters(namespace).Get(namespace, metav1.GetOptions{})
+			_, err := h.k8shelper.RookClientset.CephV1beta1().Clusters(namespace).Get(namespace, metav1.GetOptions{})
```

Comment:
how do the tests even work today in master? i wonder if this could be some source of instability

Reply:
Before this change a filesystem was not deleted when the delete was triggered on the FSClient, we could use the `RookClientSet.Ceph()` func but the func has a comment noting that it is deprectated.
Should I leave as it I changed now?

Reply:
Go ahead with the change, it's the right thing. I'll just have to deal with a conflict to change it to `CephV1()` in my branch